docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -13,6 +13,29 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.7.2 - July 4, 2024
+
+### New features
+
+- Support for running [Assess Migration](../migrate/assess-migration/) against Oracle source database.
+
+### Enhancements
+
+- In all migration workflows, indexes and triggers will now be imported to the target YugabyteDB by default in  `import-schema` step , as opposed to `import-schema --post-snapshot-import true`. This is because experiments have shown that pre-creating indexes and then importing data is faster, than importing data and then backfilling the indexes.
+
+- Modified and enhanced the call-home diagnostic functionality. More details are collected such as, source database size, migration type (offline/live), assessment report details (unsupported features/datatypes, source index/table statistics, and so on). Furthermore, call-home diagnostic details are also sent in bulk data import and live migration workflows.
+
+- More details are being sent to [yugabyted](../../reference/configuration/yugabyted/) control plane from voyager. For example, source database size details (such as total database size, total index size, total table size, total table row count), assessment report target recommendations (total colocated size, total sharded size), analyze-schema issues, and so on.
+
+- Enhanced the cluster sizing recommendation logic in assessment report:
+
+  - Minimum recommendation for node cores is now 4.
+  - Improved recommendation logic based on experiments.
+
```

Comment:
One more enhancement: 
- Optimised execution time of `get data-migration-report` by 90% for large databases.